### PR TITLE
Create a .deb package.

### PR DESCRIPTION
**Pull request by ericsnowcurrently**

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Source: txjuju
+Maintainer: Canonical Landscape Team <landscape-team@canonical.com>
+Section: python
+Priority: optional
+Build-Depends: dh-python, python-setuptools (>= 0.6b3), python-all (>= 2.6.6-3), debhelper (>= 9)
```
Comment (fginther) [inline debian/control]: I had to update the build-depends to get this to build with sbuild:

```
diff --git a/debian/control b/debian/control
index beebca6..be038c5 100644
--- a/debian/control
+++ b/debian/control
@@ -2,7 +2,16 @@ Source: txjuju
 Maintainer: Canonical Landscape Team <landscape-team@canonical.com>
 Section: python
 Priority: optional
-Build-Depends: dh-python, python-setuptools (>= 0.6b3), python-all (>= 2.6.6-3), debhelper (>= 9)
+Build-Depends: debhelper (>= 9),
+               dh-python,
+               python-all (>= 2.6.6-3),
+               python-fixtures,
+               python-setuptools (>= 0.6b3),
+               python-testtools,
+               python-twisted-core (>= 16.0.0),
+               python-twisted-names (>= 16.0.0),
+               python-twisted-web (>= 16.0.0),
+               python-yaml
 Standards-Version: 3.9.2
 Homepage: https://github.com/juju/txjuju
```

It also helps to keep this list sorted and one per line.

File: debian/control [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+Package: python-txjuju
+Architecture: any
+Depends: ${python:Depends}, ${misc:Depends}, ${extra:Depends},
+         ${shlibs:Depends},
+         python-twisted-core (>= 16.0.0),
+         python-twisted-web (>= 16.0.0),
+         python-twisted-names (>= 16.0.0),
+         python-yaml,
+         python-testtools,
+         python-fixtures
```
Comment (fginther) [inline debian/control]: It looks like python-fixtures and python-testtools are only test, and therefore build dependencies. These don't need to be clean-up now if this complicates setup.py.

File: debian/changelog [hunk 3]
```diff
@@ -0,0 +1,5 @@
+txjuju (0.9.0a1-1~ppa1) trusty; urgency=low
```
Comment (fginther) [inline debian/changelog]: "~ppa1" can be removed.

File: debian/control (hunk 1 above)
Comment (ericsnowcurrently) [inline debian/control]: fixed

**Comment by ericsnowcurrently** [comment]
I've addressed all the review comments.

**Pull request merged**